src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
 // write code here
+const list = document.querySelectorAll('li');
+
+function getEmployees(employeesList) {
+  const employees = [];
+
+  employeesList.forEach((employe) => {
+    employees.push({
+      name: employe.textContent.trim(),
+      position: employe.getAttribute('data-position'),
+      salary: employe.getAttribute('data-salary'),
+      age: employe.getAttribute('data-age'),
+    });
+  });
+
+  return employees;
+}
+
+getEmployees(list);
+
+function sortList(employeesList) {
+  const listCopy = [...employeesList];
+
+  const salaryNum = (salary) => salary.slice(1).split(',').join('');
```

Comment:
The `salaryNum` function should convert the salary string to a number. Consider using `parseFloat` or `parseInt` after removing the currency symbol and commas.